examples/single-deployment-unit-example/src/main/java/io/miragon/miranum/platform/example/adapter/out/email/EmailAdapter.java

```diff
@@ -0,0 +1,56 @@
+package io.miragon.miranum.platform.example.adapter.out.email;
+
+import io.miragon.miranum.platform.example.application.domain.SendNotification;
+import io.miragon.miranum.platform.example.application.port.out.SendNotificationOutPort;
+import jakarta.mail.Message;
+import jakarta.mail.MessagingException;
+import jakarta.mail.internet.InternetAddress;
+import jakarta.mail.internet.MimeMessage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Component;
+
+import java.util.Properties;
+
+@Slf4j
+@Component
+public class EmailAdapter implements SendNotificationOutPort {
+
+    private final JavaMailSender mailSender;
+
+    public EmailAdapter() throws jakarta.mail.MessagingException {
+        final JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        mailSender.setHost("localhost");
```

Comment:
should: would use application.yaml config. Would offer props for username and password as well